soccer/gameplay/tactics/positions/submissive_defender.py

```diff
@@ -151,6 +199,29 @@ def execute_marking(self):
     def on_exit_marking(self):
         self.remove_subbehavior('move')
 
+    def on_enter_clearing(self):
+        print("CLEARING BALL")
+        #copied from submissivegoalie
+        kick = skills.pivot_kick.PivotKick()
+
+        kick.aim_params['error_threshold'] = 1.0
+        kick.aim_params['max_steady_ang_vel'] = 12
+
+        # chip
+        kick.chip_power = 1.0
+        kick.use_chipper = True
+
+        kick.target = robocup.Segment(
+            robocup.Point(-constants.Field.Width / 2, constants.Field.Length),
+            robocup.Point(constants.Field.Width / 2, constants.Field.Length))
+
+        self.add_subbehavior(kick, 'kick-clear', required=True)
```

Comment:
Due to a limitation in our role assignment system (https://github.com/RoboJackets/robocup-software/issues/90), `required` should be set to False here.  Basically the issue is that the Defender behavior is optional (required=False), but it's optionalness doesn't carry down to its subbehaviors.  It's on the todo-list to make the `required` flag make more sense in the context of parent/child behaviors, but it hasn't been done yet.